packages/fx-core/src/component/driver/deploy/azure/impl/baseDeployImpl.ts

```diff
@@ -75,27 +78,6 @@ export abstract class BaseDeployImpl {
     return path.isAbsolute(inputPath) ? inputPath : path.join(baseFolder, inputPath);
   }
 
-  /**
-   * pack dist folder into zip
-   * @param args dist folder and ignore files
-   * @param context log provider etc..
-   * @protected
```

Comment:
This function is just replaced without any changes, right?

Reply:
Yes, just moved those function to other class.